board/piksiv3/rootfs-overlay/etc/init.d/S75dev_boot

```diff
@@ -16,7 +16,11 @@ copy_from_sd() {
   cp "$sd_file" "$output_file"
 
   if [ ! -f "$output_file" ]; then
-    echo "ERROR: copy failed"
+    while [ 1 ]; do
+      echo "ERROR: copy of images from SD card failed. Please Reboot"
+      echo "ERROR: copy of images from SD card failed. Please Reboot" | SBP_LOG --error
```

Comment:
ZMQ infrastructure will not be up when this script executes, so the SBP log stuff will not be visible and possibly hang.